docs/lib/datastore/fragments/ios/getting-started/60_saveSnippet.md

```diff
@@ -0,0 +1,12 @@
+```swift
+let post = Post(title: "Create an Amplify DataStore app")
+
+Amplify.DataStore.save(post) { result in
+    switch result {
+    case .success:
+        print("Post saved successfully!")
```

Comment:
Similar to Android, this is when is stored in device storage not synced, it would be useful to make that distinction?